### DISCUSSION TITLE
🔮 ✨🍂 What future awaits you! Welcome to Education Study Breaks November edition! 🎓 💬 🧑‍🎓

### DISCUSSION THREAD

**Discussion by samus-aran** [community/community#143876] [GitHub Education]
It’s that time again - **GitHub Education Study Breaks 📖!** 🗳️ Every month, we’re answering a fun question with the help of some good ol’ emojis. Let’s see where the community stands! 😎✨

**This month’s question:**

### What do you want to be when you finish your studies?

Vote with an emoji that matches your passion! If your option isn’t listed, just drop it in the comments – we wanna hear what you inspire to be and how you are achieving it! ⛰️  💬  The best ones will make it into our next community check-in and student newsletter! 🙌 📧

Here are the options:

- 👀 Software Developer
- ❤️ AI Engineer
- 🎉 Game Developer
- 🚀 Still figuring it out!
- 👍 Other

Take one step at a time and sooner than you realise that mountain you're climbing won't seem too daunting ⛰️ 💪 

Let the votes roll in, and let’s see what amazing careers our community want to work towards ✨ 🚀

**Comment by FeiSuoerDe** [comment]
🎉

**Reply by samus-aran** [reply]
What game engine are you using? 🎮

**Reply by FeiSuoerDe** [reply]
godot and unity

**Reply by samus-aran** [reply]
Nice! What you currently playing at the moment? I just don't have time like I used to so I'm always re-visiting my island on Animal Crossing 🐱

**Reply by FeiSuoerDe** [reply]
> Nice! What you currently playing at the moment? I just don't have time like I used to so I'm always re-visiting my island on Animal Crossing 🐱

rimworld,and look this[https://github.com/[FeiSuoerDe/development](https://github.com/FeiSuoerDe/development)]

**Comment by darshanbagade** [comment]
👀

**Reply by samus-aran** [reply]
You gotta tell us what languages you are focusing on! If you could work in any tech company which would it be? 👀

**Reply by darshanbagade** [reply]
Java  or Javascript

**Reply by darshanbagade** [reply]
And what is your option from your post ?
👀 Software Developer
❤️ AI Engineer
🎉 Game Developer
🚀 Still figuring it out!
👍 Other

**Reply by samus-aran** [reply]
My background is Software Development 😅 Then I progressed into community management because I love working with folks and helping them achieve things 💪

**Reply by darshanbagade** [reply]
That is excellent work 👏.

**Comment by melqtx** [comment]
👀 oscillating bw ❤️

**Reply by samus-aran** [reply]
whaaaaaaaat is that?! 🤔

**Reply by darshanbagade** [reply]
It means prompt engineering. 😂
